app/src/main/res/values/strings.xml

```diff
@@ -87,185 +87,186 @@
-    <string name="create_from_string_hint">Puzzle</string>
-    <string name="create_incorrect_puzzle">Incorrect puzzle</string>
-    <string name="multiple_solution_text">This puzzle has more than one solution. The correct sudoku puzzle must have a unique solution</string>
-    <string name="no_solution_text">This puzzle has no solutions</string>
+    <string name="create_from_string_text">Enter a sudoku in string form. Empty cells are indicated by a \".\" or a \"0\".</string>
+    <string name="create_from_string_hint">Sudoku</string>
+    <string name="create_incorrect_puzzle">Incorrect sudoku</string>
+    <string name="multiple_solution_text">More than one solution. Proper sudokus must have a unique solution.</string>
+    <string name="no_solution_text">No solutions</string>
     <string name="create_import_set">Set</string>
-    <string name="custom_sudoku_no_added">You haven\'t added any sudokus yet</string>
+    <string name="custom_sudoku_no_added">No sudokus added yet</string>
     <string name="custom_sudoku_delete_dialog_title">Delete selected</string>
-    <string name="custom_sudoku_delete_dialog_text">Are you sure you want to delete the selected sudokus? This action cannot be undone</string>
+    <string name="custom_sudoku_delete_dialog_text">Delete the selected sudokus permanently?</string>
     <string name="game_not_started">Not started</string>
-    <string name="export_string_text">Select text from the text field or simply click the copy button</string>
+    <string name="export_string_text">Select text from the text field, or click the \"Copy\" button.</string>
     <string name="export_string_title">Export as string</string>
     <string name="export_string_copy">Copy</string>
     <string name="export_string_state_copied">Copied</string>
     <string name="export_string_state_error">Error</string>
     <string name="filter_all">All</string>
     <string name="filter_completed">Completed</string>
-    <string name="filter_in_progress">In progress</string>
+    <string name="filter_in_progress">Underway</string>
     <string name="filter_not_started">Not started</string>
     <string name="sort_ascending">Ascending</string>
     <string name="sort_descending">Descending</string>
     <string name="sort_by_timer">Timer</string>
     <string name="sort_by_game_id">ID</string>
     <string name="sort_label">Sort</string>
     <string name="filter_label">Filter</string>
-    <string name="help_translate">Help translate</string>
+    <string name="help_translate">Translation</string>
     <string name="title_folders">Folders</string>
     <string name="create_folder">Create a folder</string>
     <string name="delete_folder">Delete a folder</string>
     <string name="folder_name">Folder: %1$s</string>
     <plurals name="number_puzzles_to_import">
-        <item quantity="one">%1$d puzzle to import</item>
```

Comment:
Tried to get rid of all puzzle/s, as this is often mistaken for "puzzle-piece puzzles" in other languages, and subsequent translations.